src/NodaTime/Period.cs

```diff
@@ -424,14 +424,14 @@ public static int DaysBetween(LocalDate start, LocalDate end)
         }
 
         /// <summary>
-        /// Returns the difference between start and end time in nanoseconds.
+        /// Returns the number of nanoseconds between two <see cref="LocalTime"/> objects.
```

Comment:
```suggestion
        /// Returns the number of nanoseconds between two <see cref="LocalTime"/> values.
```

(Sorry, didn't clock the word "objects" before, which isn't appropriate when LocalTime is a struct...)